### PR TITLE
Jexl evaluation stability

### PR DESCRIPTION
**Pull request by winged**
Various fixes and improvements in question JEXL evaluation

### CODE REVIEW

File: caluma/form/tests/test_validators.py [hunk 1]
```diff
@@ -104,3 +104,74 @@ def test_validate_nested_form(
...
+    ],
+)
+def test_validate_invalid_jexl(
+    db,
+    form_question,
+    document,
+    answer,
+    question,
+    document_factory,
+    answer_factory,
```
Comment (czosel) [inline caluma/form/tests/test_validators.py]: Unused factories

Comment (winged) [reply caluma/form/tests/test_validators.py]: Fixed

**Comment by winged** [comment]
Rebased, fixed some merge problems. Please recheck @open-dynaMIX @czosel

**Comment by czosel** [comment]
@winged travis was timing out, I just hit "retry"

**Pull request merged**